docs/submission/validating-molecular-data.md

```diff
@@ -0,0 +1,447 @@
+pip3 install . # installs seq-tools
+```
+
+You can verify that the installation worked by checking the version: 
+```bash
+seq-tools -v
+```
+To update the client to the latest version, pull the latest version of the repository and run: 
+```bash
+cd seq-tools
+pip3 install .
+```
+
+</TabItem>
+<TabItem value="docker">
+
+```bash
+docker pull quay.io/icgc-argo/seq-tools:1.0.0
+alias seq-tools-in-docker="docker run -t -v `pwd`:`pwd` -w `pwd` quay.io/icgc-argo/seq-tools:1.0.0 seq-tools"
+```
+
+</TabItem>
+</Tabs>
+
+<!---  Tabs end here -->
+
+
+### Structured Directory Validation
+One method to quickly complete validation is to structure your data in a pre-determined format that `seq-tools` is expecting. Under a submission directory, compile each submission as a folder, with the JSON payload and the associated data files in the same folder. 
+
+Each submission folder **must** contain:
+- The metadata payload as a `.json` file. Only 1 `.json` file must be present in the submission folder.
+- All molecular data files associated to that payload, or symlinks to their location.
+
+For example: 
+![Example Directory Structure](/assets/submission/molecular_submission_validation_directory.png)
+
+Once your data is ready, use `seq-tools` from the submission directory to validate all of the data: 
+
+<!---  Tabs start here -->
+
+<Tabs
+  groupId="operating-systems"
+  defaultValue="client"
+  values={[
+    { label: 'Client', value: 'client', },
+    { label: 'Docker', value: 'docker', },
+  ]
+}>
+<TabItem value="client">
+
+```bash
+seq-tools validate submission-directory/*/*.json  
+```
+
+</TabItem>
+<TabItem value="docker">
+
+```bash
+cd ..
+seq-tools-in-docker validate -d submission-directory/*/*.json 
+```
+
+</TabItem>
+</Tabs>
+
+<!---  Tabs end here -->
+
+
+You will see an interactive prompt of the results of validation as the tool runs.  Once completed, view the details of the validation report:
+
+```bash
+cat validation_report.PASS.jsonl | jq . | less
+cat validation_report.INVALID.jsonl | jq . | less
+```
+### Scriptable Location Validation
+
+If you do not want to use the structured directory, you can use the '-d' option if data files are located in a different directory than where the metadata payloads are listed.
+
+```bash
+seq-tools validate -d ../sequencing-data-directory/ metadata_file_only/sequencing_experiment_payload.json
+```
+Or validate all metadata payloads in a directory at once using a wildcard: 
+```
+seq-tools validate -d ../sequencing-data-directory/ submission-payload-directory/*/*.json  
+seq-tools-in-docker validate -d ../sequencing-data-directory/ submission-payload-directory/*/*.json
+```
+
+You can use this method to build your own script to validate payloads in different locations. 
+
+### Understanding the Validation Report
+Validation report summaries can be found in the top level submission directory where you ran `seq-tools`. Reports are separated into three categories:
```

Comment:
---> Validation report summaries can be found in folder where you ran `seq-tools`.